### PR TITLE
OBPIH-4207,4208 Add option to show only errored items on the edit page

### PR DESCRIPTION
**Pull request by awalkowiak**

### CODE REVIEW

File: src/js/components/form-elements/TableBodyVirtualized.jsx [hunk 1]
```diff
@@ -38,7 +38,14 @@ class TableBodyVirtualized extends Component {
       }
     } else {
       _.forEach(fields.value, (field) => {
+        const dynamicAttr = getDynamicRowAttr ?
+          getDynamicRowAttr({ ...properties, rowValues: field }) : {};
         const subfields = field[subfieldKey];
+
+        if (dynamicAttr.hideRow) {
+          return;
```
Comment (pmuchowski) [inline src/js/components/form-elements/TableBodyVirtualized.jsx:46]: I think you should return 0 here

File: src/js/components/form-elements/TableBodyVirtualized.jsx [hunk 2]
```diff
@@ -88,6 +99,16 @@ class TableBodyVirtualized extends Component {
     const { totalCount } = properties;
 
     if (fields.value[index]) {
+      const dynamicRowAttr = fieldsConfig.getDynamicRowAttr ?
+        fieldsConfig.getDynamicRowAttr({
+          ...properties,
+          rowValues: _.filter(fields.value, v => !v.hideRow)[index],
```
Comment (pmuchowski) [inline src/js/components/form-elements/TableBodyVirtualized.jsx:105]: Why are you filtering the fields here?

**Review by pmuchowski — CHANGES_REQUESTED** [review]

File: src/js/components/form-elements/TableBodyVirtualized.jsx (hunk 1 above)
Comment (awalkowiak) [reply src/js/components/form-elements/TableBodyVirtualized.jsx:46]: @pmuchowski this is lodash's forEach version of `continue`

File: src/js/components/form-elements/TableBodyVirtualized.jsx (hunk 2 above)
Comment (awalkowiak) [reply src/js/components/form-elements/TableBodyVirtualized.jsx:105]: I think this is something, that I forgot to remove. I'll check it.

**Pull request merged**